CMakeLists.txt

```diff
@@ -167,6 +168,9 @@ endif(MSVC)
 
 if (ANDROID OR IOS)
   set(BUILD_ATEN_MOBILE ON)
+  if (BUILD_TORCH_MOBILE)
+    set(TORCH_MOBILE ON)
```

Comment:
you set this var - do you use it anywhere?

Reply:
@dzhulgakov it is the source code level control to not risk regressing internal facebook product, see discussion: https://fburl.com/hyk9n7c2
It's used by stacked diff(PR).